### PR TITLE
documentation(EJ2-67529): Added chart-combinedYAxis sample.

### PR DESCRIPTION
**Pull request by Yash58vanth**
Added chart-combinedYAxis sample and updated showMemberSeries.

### CODE REVIEW

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/pivot-chart.md [hunk 1]
```diff
@@ -624,7 +657,7 @@ Furthermore, with a single click over the legend item, you can show or hide spec
 
 
 
-![Chart series displayed with same colors for same points/members](images/showMemberSeries.png)
+![PivotChart with Show member based chart series](images/showMemberSeries.png)
```
Comment (SasthaPrathap) [inline ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/pivot-chart.md:660]: `Show point color based on members in Pivot Chart`

**Review by SasthaPrathap — CHANGES_REQUESTED** [review]
Check and clear the review feedbacks.

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/pivot-chart.md (hunk 1 above)
Comment (SasthaPrathap) [reply ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/pivot-chart.md:660]: Image look too small. Take image with 720 X 420 px. Tittle not required.

**Review by SasthaPrathap — CHANGES_REQUESTED** [review]
Check and clear the review feedbacks.

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/pivot-chart.md (hunk 1 above)
Comment (Yash58vanth) [reply ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/pivot-chart.md:660]: cleared

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.MVC/pivot-chart.md [hunk 2]
```diff
@@ -591,11 +591,44 @@ In the following code sample, the pivot chart can be seen as a single chart with
...
+{% endhighlight %}
+{% highlight c# tabtitle="combinedYAxis.cs" %}
+{% include code-snippet/pivot-table/pivot-chart/single-combined-y-axis/combinedYAxis.cs %}
+{% endhighlight %}
+{% endtabs %}
+{% endif %}
+
+
+
+![output](images/single-combined-y-axis.png)
```
Comment (SasthaPrathap) [inline ej2-asp-core-mvc/pivot-table/EJ2_ASP.MVC/pivot-chart.md:625]: add the image caption `Pivot Chart for multiple values in a single y-axis`

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.MVC/pivot-chart.md [hunk 3]
```diff
@@ -622,7 +655,7 @@ Furthermore, with a single click over the legend item, you can show or hide spec
 
 
 
-![Chart series displayed with same colors for same points/members](images/showMemberSeries.png)
+![PivotChart with Show member based chart series](images/showMemberSeries.png)
```
Comment (SasthaPrathap) [inline ej2-asp-core-mvc/pivot-table/EJ2_ASP.MVC/pivot-chart.md:658]: `Show point color based on members in Pivot Chart`

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/pivot-chart.md [hunk 4]
```diff
@@ -593,11 +593,44 @@ In the following code sample, the pivot chart can be seen as a single chart with
...
+{% endhighlight %}
+{% highlight c# tabtitle="combinedYAxis.cs" %}
+{% include code-snippet/pivot-table/pivot-chart/single-combined-y-axis/combinedYAxis.cs %}
+{% endhighlight %}
+{% endtabs %}
+{% endif %}
+
+
+
+![output](images/single-combined-y-axis.png)
```
Comment (SasthaPrathap) [inline ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/pivot-chart.md:627]: `Pivot Chart for multiple values in a single y-axis`

**Review by SasthaPrathap — CHANGES_REQUESTED** [review]
Check and clear the review feedbacks.

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.MVC/pivot-chart.md [hunk 5]
```diff
@@ -591,11 +591,44 @@ In the following code sample, the pivot chart can be seen as a single chart with
...
+{% endtabs %}
+{% endif %}
+
+
+
+![output](images/single-combined-y-axis.png)
+
+### Show point color based on members
+
+When multiple axes are enabled, you can display the same color for each member in the column axis by setting the [`showPointColorByMembers`](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.PivotView.PivotViewChartSettings.html#Syncfusion_EJ2_PivotView_PivotViewChartSettings_ShowPointColorByMembers) property to **true** in the [`chartSettings`](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.PivotView.PivotViewChartSettings.html). As a result, the end user can easily identify each member across different measures in the entire chart.
```
Comment (SasthaPrathap) [inline ej2-asp-core-mvc/pivot-table/EJ2_ASP.MVC/pivot-chart.md:629]: APIs are not related to MVC standard

**Review by SasthaPrathap — CHANGES_REQUESTED** [review]
Clear the review feedbacks.

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.MVC/pivot-chart.md (hunk 5 above)
Comment (Yash58vanth) [reply ej2-asp-core-mvc/pivot-table/EJ2_ASP.MVC/pivot-chart.md:629]: cleared

**Review by SasthaPrathap — APPROVED** [review]

**Pull request merged**